web/js-quickstart/src/index.js

```diff
@@ -50,27 +54,35 @@ function createElementWithClass(tag, className) {
 function renderPeer(peer) {
   const peerTileDiv = createElementWithClass("div", "peer-tile");
   const videoElement = createElementWithClass("video", "peer-video");
-  const peerTileName = createElementWithClass("span", "peer-name");
+  const peerTileName = createElementWithClass("div", "peer-name");
   videoElement.autoplay = true;
   videoElement.muted = true;
   videoElement.playsinline = true;
   peerTileName.textContent = peer.name;
-  hmsActions.attachVideo(peer.videoTrack, videoElement);
+
+  hmsStore.subscribe((track) => {
+    if (!track || !videoElement) {
+      return;
+    }
+    hmsActions.attachVideo(track.id, videoElement);
+  }, selectVideoTrackByID(peer.videoTrack));
```

Comment:
why is this subscription needed?

Reply:
peer.videoTrack is not available and we only subscribe  once.